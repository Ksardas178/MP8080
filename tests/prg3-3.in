/*Рассмотрите сложение двоично-десятичных
чисел. Рассмотрите действие команды DAA 
для различных значений суммируемых операндов.
Результат выполнения фиксируйте в памяти.*/

//===========================================(36,27)
MVI B 36
MVI C 27

//init
MOV A C
MVI D 0
MVI E 0









