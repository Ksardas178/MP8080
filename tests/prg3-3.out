
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: MVI 0 36
00000001: MVI 1 27
00000010: MOV 7 1
00000011: MVI 2 0
00000100: MVI 3 0
